WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -136,7 +136,13 @@ public void onResponse(JSONObject response) {
                         mRemoteSettings.localTableId = mBlog.getRemoteBlogId();
                         deserializeDotComRestResponse(mBlog, response);
                         if (!mRemoteSettings.equals(mSettings)) {
+                            // postFormats setting is not returned by this api call so copy it over
```

Comment:
Ah no, that's good, I've been ticked by the name "SiteSettingsInterface". Also, I'm not sure if it's related to your patch but my post format list remains empty. I'll have a look at the code.